### ISSUE TITLE
SOX not working on Stack Overflow when logged out

### ISSUE THREAD

**Issue by mlaunois** [soscripted/sox#423]
## Description
The Stack Overflow Extras do not load when I'm logged out on [stackoverflow.com](https://stackoverflow.com). It does work on every other SE site.

## Steps to reproduce
1. [Log out](https://stackoverflow.com/users/logout) from Stack Overflow
2. Confirm the logout:
![Logging out from Stack Overflow](https://user-images.githubusercontent.com/47182594/63226062-88dfab80-c1d7-11e9-9c8f-b4b8848b5bef.png)
3. The SOX settings button is missing from the topbar.

## SOX errors logged in console
```
Uncaught SOX: There was an error while attempting to initialize the SOX Settings Dialog, please report this on GitHub.
TypeError: Cannot read property 'left' of undefined
```

## Expected behaviour
The SOX settings button should appear on the Stack Overflow topbar, like on Meta Stack Overflow:
![Meta Stack Overflow topbar](https://user-images.githubusercontent.com/47182594/63226285-6c903e80-c1d8-11e9-9b70-3907ad53a4ed.png)

## Screenshots
![Screenshot 1](https://user-images.githubusercontent.com/47182594/63225905-1cb07800-c1d6-11e9-8024-c9441fb0b4b4.png)

## Environment
SOX version: 2.5.23 DEV
Platform: Violentmonkey
Browser: Chrome 64-bit 74.0.3729.108

---

## Features enabled 
```js
[
  "Appearance-addAuthorNameToInboxNotifications",
  "Appearance-alignBadgesByClass",
  "Appearance-answerTagsSearch",
  "Appearance-colorAnswerer",
  "Appearance-displayName",
  "Appearance-dragBounty",
  "Appearance-highlightQuestions",
  "Appearance-isQuestionHot",
  "Appearance-localTimestamps",
  "Appearance-markEmployees",
  "Appearance-metaChatBlogStackExchangeButton",
  "Appearance-metaNewQuestionAlert",
  "Appearance-scrollToTop",
  "Appearance-spoilerTip",
  "Appearance-standOutDupeCloseMigrated",
  "Appearance-tabularReviewerStats",
  "Appearance-topAnswers",
  "Appearance-unspoil",
  "Appearance-addTimelineAndRevisionLinks",
  "Appearance-showTagWikiLinkOnTagPopup",
  "Appearance-hideWelcomeBackMessage",
  "Appearance-hideHowToAskWhenZoomed",
  "Comments-autoShowCommentImages",
  "Comments-commentReplies",
  "Comments-commentShortcuts",
  "Comments-confirmNavigateAway",
  "Comments-copyCommentsLink",
  "Comments-moveBounty",
  "Comments-showCommentScores",
  "Comments-hiddenCommentsIndicator",
  "Comments-onlyShowCommentActionsOnHover",
  "Editing-addSBSBtn",
  "Editing-editComment",
  "Editing-editReasonTooltip",
  "Editing-findAndReplace",
  "Editing-kbdAndBullets",
  "Editing-titleEditDiff",
  "Editing-inlineEditorEverywhere",
  "Flags-flagOutcomeTime",
  "Flags-flagPercentages",
  "Flags-flagPercentageBar",
  "Sidebar-hideCommunityBulletin",
  "Sidebar-hideJustHotMetaPosts",
  "Sidebar-hideHireMe",
  "Sidebar-hideChatSidebar",
  "Sidebar-hideLoveThisSite",
  "Sidebar-linkedToFrom",
  "Sidebar-hotNetworkQuestionsFiltering",
  "Chat-chatEasyAccess",
  "Chat-replyToOwnChatMessages",
  "Chat-renameChat",
  "Voting-betterCSS",
  "Voting-stickyVoteButtons",
  "Voting-disableVoteButtons",
  "Extras-linkedPostsInline",
  "Extras-parseCrossSiteLinks",
  "Extras-quickAuthorInfo",
  "Extras-shareLinksPrivacy",
  "Extras-shareLinksMarkdown",
  "Extras-sortByBountyAmount",
  "Extras-warnNotLoggedIn",
  "Extras-hideCertainQuestions",
  "Extras-showMetaReviewCount",
  "Extras-copyCode",
  "Extras-dailyReviewBar",
  "Extras-openLinksInNewTab",
  "Extras-showQuestionStateInSuggestedEditReviewQueue",
  "Extras-addOnTopicLinkToSiteSwitcher",
  "Extras-customMagicLinks"]
```

**Comment by shu8** [comment]
Thanks for the bug report! Unfortunately I won't be able to look into this
until a few weeks. I'd be happy to review a pull request if you're able to
submit one but otherwise it may be a few weeks till I get time!

**Comment by double-beep** [comment]
I have confirmed the issue and worked on a fix which is in PR 424. The PR is currently awaiting review.

**Comment by shu8** [comment]
This should be fixed in [dev 2.5.26](https://github.com/soscripted/sox/raw/dev/sox.user.js) now thanks to @double-beep's PR :)

**Comment by mlaunois** [comment]
@doublebeep OK, it works again
